main.rb

```diff
@@ -0,0 +1,76 @@
+require 'open3'
+require 'os'
+
+def get_env_variable(key)
+  return (ENV[key] == nil || ENV[key] == "") ? nil : ENV[key]
+end
+
+def env_has_key(key)
+  value = get_env_variable(key)
+	return value unless value.nil? || value.empty?
+
+	abort("Input #{key} is missing.")
```

Comment:
It should be same indentation:

```suggestion
  value = get_env_variable(key)
  return value unless value.nil? || value.empty?

  abort("Input #{key} is missing.")
```